src/styles/components/top-bar.css

```
@@ -0,0 +1,171 @@
+}
+
+.auth-component > button {
+  /* Transitions handled automatically by layout/grid.css */
+
+  /* Modern @starting-style animation */
+  @starting-style {
+    opacity: 0;
+    transform: translateX(10px);
+  }
+
+  opacity: 1;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

opacity is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)